pkg/idmap/idmapped_utils.go

```diff
@@ -61,11 +61,11 @@ func CreateUsernsProcess(uidMaps []idtools.IDMap, gidMaps []idtools.IDMap) (int,
 		_ = unix.Prctl(unix.PR_SET_PDEATHSIG, uintptr(unix.SIGKILL), 0, 0, 0)
 		// just wait for the SIGKILL
 		for {
-			syscall.Pause()
+			_ = syscall.Pause()
 		}
 	}
 	cleanupFunc := func() {
-		unix.Kill(int(pid), unix.SIGKILL)
+		_ = unix.Kill(int(pid), unix.SIGKILL)
```

Comment:
This should probably warn on unexpected errors like `EPERM` (but not `ESRCH`). (Or maybe outright bail out, because `Wait4` might hang otherwise. But that might be a separate PR.)